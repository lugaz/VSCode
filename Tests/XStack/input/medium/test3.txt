push 895
peek
peek
push 271
peek
peek
peek
peek
pop
pop
push 205
peek
peek
pop
peek
pop
peek
push 648
push 553
peek
pop
push 45
peek
push 569
push 144
peek
peek
pop
peek
peek
pop
peek
push 958
pop
peek
peek
push 763
peek
push 6
peek
peek
push 191
peek
push 503
push 474
pop
push 237
pop
pop
peek
peek
pop
pop
pop
pop
push 608
push 573
pop
peek
peek
pop
peek
push 619
peek
push 998
pop
pop
peek
push 328
pop
push 243
peek
push 310
push 652
pop
pop
pop
peek
peek
push 21
pop
push 886
peek
push 376
peek
pop
pop
peek
push 82
peek
pop
pop
peek
push 206
pop
push 12
push 64
pop
push 792
peek
pop
push 814
push 446
push 279
peek
push 332
push 675
push 18
peek
peek
peek
pop
peek
peek
push 8
pop
