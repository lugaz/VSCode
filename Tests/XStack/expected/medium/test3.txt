895
895
271
271
271
271
271
895
205
205
205
-1
-1
-1
553
553
45
144
144
144
569
569
569
45
958
45
45
763
6
6
191
474
237
503
191
191
191
6
763
45
573
608
608
608
648
619
998
619
648
328
243
652
310
243
648
648
21
886
376
376
886
648
82
82
648
-1
206
64
792
792
279
18
18
18
18
675
675
8
